src/EntityFramework/Core/Query/PlanCompiler/NestPullup.cs

```diff
@@ -1262,7 +1269,6 @@ private Node ProjectOpCase2(Node projectNode)
 
             Command.RecomputeNodeInfo(projectNode);
             var projectNodeInfo = Command.GetExtendedNodeInfo(projectNode);
-            var nestNodeInfo = Command.GetExtendedNodeInfo(nestNode);
```

Comment:
Maybe, but it wasn't inside a DEBUG section. And a debugger could run that expression in the Immediate window if they needed it - so don't suggest putting this back in.